ext/puma_http11/mini_ssl.c

```diff
@@ -339,20 +344,6 @@ sslctx_initialize(VALUE self, VALUE mini_ssl_ctx) {
     SSL_CTX_set_verify(ctx, NUM2INT(verify_mode), engine_verify_callback);
   }
 
-  // Random.bytes available in Ruby 2.5 and later, Random::DEFAULT deprecated in 3.0
-  session_id_bytes = rb_funcall(
-#ifdef HAVE_RANDOM_BYTES
-    rb_cRandom,
-#else
-    rb_const_get(rb_cRandom, rb_intern_const("DEFAULT")),
-#endif
-    rb_intern_const("bytes"),
-    1, ULL2NUM(SSL_MAX_SSL_SESSION_ID_LENGTH));
-
-  SSL_CTX_set_session_id_context(ctx,
-                                 (unsigned char *) RSTRING_PTR(session_id_bytes),
-                                 SSL_MAX_SSL_SESSION_ID_LENGTH);
-
```

Comment:
Something we no longer need?

Reply:
I don't think so, but I thought I'd sit on it for a bit...

Reply:
I was wrong.  Removing it causes issues with TLSv1.3 connections.  Kind of odd.  Regardless, I added the code back in...